### ISSUE TITLE
Heroic Sethekk Halls lockout continually extending itself

### ISSUE THREAD

**Issue by krellemeister** [dalaranwow/dalaran-wow#3104]
After defeating the Raven Lord for the Swift Flight Form druid questline, I got, as usual, a lockout period on the dungeon. After going back the next day to try my luck once again on getting the mount, I noticed that the lock hadn't reset like the lock information for it implied. I decided to wait it out and hopefully have it go away by friday morning, which is the global raid reset day. Again it decided to automatically extend itself, so I'm now stuck with it for what seems like the rest of eternity. 

I cannot really provide more information on this matter, as I'm unsure where it went wrong myself. I thought it'd still be important to have it listed here on the bug tracker though.

**Comment by zopQQ** [comment]
I can confirm and can give some more information.

2 days after I did the instance once it still showed me that it's locked. I found it weird so I went in to see if it really is. When I entered it seemed fine, the trash was there so were the 2 bosses you find in normal as well - Darkweaver Syth and Talon King Ikiss. But when my druid decided to summon Anzu, it didn't work.

So I guess the whole problem is only Anzu not respawning, cuz the other bosses do at the heroic dungeon reset time every day (4:59 am ST).
